tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/CombinationListingFeatureContext.php

```diff
@@ -263,6 +263,9 @@ private function assertListedCombinationsProperties(array $expectedDataRows, arr
         foreach ($listCombinations as $key => $editableCombinationForListing) {
             $expectedCombination = $expectedDataRows[$key];
 
+            // @todo: assert combination images when we implement combination image association commands
+            Assert::assertNull($editableCombinationForListing->getImagePath(), 'Unexpected combination image');
```

Comment:
added todo here for next pr. When we can add image to generated combiantion, then we will be able to test it, else its always null.